ecosystem/typescript/sdk/src/aptos_client.test.ts

```diff
@@ -110,7 +110,7 @@ test(
     accountResource = resources.find((r) => r.type === aptosCoin);
     expect((accountResource!.data as any).coin.value).toBe("717");
   },
-  30 * 1000,
+  60 * 1000,
```

Comment:
Maybe defining this value in test_helper.test.ts?

Reply:
Done.